src/metabase/notification/send.clj

```diff
@@ -385,7 +386,7 @@
                      :payload_type    (:payload_type notification)}
     (let [options      (merge *default-options* options)
           notification (with-meta notification {:notification/triggered-at-ns (u/start-timer)})]
-      (log/debugf "Sending %s" (if (:notification/sync? options) "synchronously" "asynchronously"))
+      (log/debugf "Will be send %s" (if (:notification/sync? options) "synchronously" "asynchronously"))
```

Comment:
"Will be sent"

BTW, what do you think about writing both of the strings completely instead of using format? To increase greppability.